src/cls/_ZPM/PackageManager/Developer/Processor/Abstract.cls

```diff
@@ -224,6 +225,26 @@ Method %Evaluate(pAttrValue) As %String [ Internal ]
 	Quit tAttrValue
 }
 
+Method ApplyCustomParameters(pAttributes As %Library.ArrayOfDataTypes, ByRef pParams) As %Status
+{
+	Do ##class(%ZPM.PackageManager.Developer.ModuleSetting.Default).GetCustomParameters(.customParams,..ResourceReference.Module, .pParams)
+	Set key = ""
+	For {
+		Set attribute = pAttributes.GetNext(.key)
+		Quit:(key="")
+		Do pAttributes.SetAt(##class(%ZPM.PackageManager.Developer.ModuleSetting.Default).EvaluateAttribute(attribute,.customParams),key)
+		// try to set a Property
+		Try {
+			Set val = $Property($this,key)
+			Continue:($IsObject(val)||$Listvalid(val))
+			Set $Property($this,key) = ..%Evaluate(pAttributes.GetAt(key))
```

Comment:
So, you just set some else value?
$Property($this,key) had already some value, but you set there something from `pAttributes.GetAt(key)`
looks like it should change the value only in case if it found